src/utils/MemoryFS.js

```
@@ -0,0 +1,471 @@
+   * @param {string} path
+   * @param {{ recursive?: boolean, mode?: number } | number} [options]
+   */
+  async mkdir(path, options) {
+    const p = normalizePath(path)
+    const recursive =
+      options && typeof options === 'object' ? !!options.recursive : false
+    const mode =
+      options && typeof options === 'object' && options.mode != null
+        ? options.mode
+        : typeof options === 'number'
+        ? options
```

ESLint

[error] Insert `··`

(prettier/prettier)